lib/content-types.js

```diff
@@ -2,11 +2,43 @@
 var contentType = require("content-type");
 
 var html = ["text/html", "application/xml+xhtml", "application/xhtml+xml"];
+var xml=["text/xml"]
+var ts=["video/mp2t"]
+var z["application/x-7z-compressed"]
+var csv=["text/csv"]
+var ttf=["font/ttf"]
+var php=["application/x-httpd-php"]
+var doc=["application/msword"]
+var weba=["audio/webm"]
+var webp=["image/webp"]
 var css = ["text/css"];
+var rar=["application/vnd.rar"]
+var json=["application/json","text/x-json"];
+var text=["text/plain"]
+var woff=["font/woff"]
+var gz=["appliction/gzip"]
+var woff2=["font/woff2"]
+var xls=["application/vnd.ms-excel"]
+var 3g2["video/3gpp2"]
+var zip=["application/zip"]
+var epub=["application/ebup+zip"]
+var gif=["image/gif"]
+var jar=["application/java-archive"]
+var ics=["text/calender"]
+var typescript=["application/x-typescript"]
 var javascript = [
   "text/javascript",
   "application/javascript",
   "application/x-javascript",
+  "text/javascript+module",
+  "application/javascript+module",
+  "text/jsscript",
+  "text/vbscript",
+  "TeXt/jaVaScRiPt",
```

Comment:
the `content-type` library automatically lower-cases it, so this one will never be triggered.